pkg/model/message_test.go

```diff
@@ -154,17 +155,17 @@ func Test_PopulateTranslations(t *testing.T) {
 
 	tests := []struct {
 		name         string
-		messageSlice MessagesSlice
+		translations Translations
 	}{
 		{
-			// Only original messages -> noop
+			// Only original translation -> noop
 			name:         "Nothing to populate",
-			messageSlice: onlyOriginal,
+			translations: onlyOriginal,
 		},
 		{
-			// Original messages have extra messages -> translated messages should be populated with the extra messages.
+			// Original translation have extra messages -> translated messages should be populated with the extra messages.
```

Comment:
// Original translation has extra messages -> translated messages should be populated with the extra messages.